src/components/screen/desktop.js

```diff
@@ -419,28 +420,32 @@ export class Desktop extends Component {
         this.setState({ showNameBar: true });
     }
 
+    addFolder = (folder_name) => {
```

Comment:
change this function name to "addToDesktop", for future confusion!

Reply:
Okay .... I will do the nesessary changes and commit